pkgs/applications/audio/littlegptracker/default.nix

```
@@ -1,75 +1,63 @@
 { lib, stdenv
 , fetchFromGitHub
-, unstableGitUpdater
-, SDL
+, SDL2
+, python3
 , jack2
 , Foundation
+, alsa-lib
+, pkg-config
 }:
-
-stdenv.mkDerivation {
+let
+  python = python3.withPackages (ps: with ps; [ pillow ]);
+  platform = if stdenv.hostPlatform.isLinux then "X64" else "OSX";
```

```suggestion
  platform = if stdenv.hostPlatform.isDarwin then "OSX" else "X64";
```

This way non-linux non-darwin platforms will use X64